src/access/tests/BUILD.gn

```diff
@@ -18,12 +18,17 @@ import("//build_overrides/nlunit_test.gni")
 
 import("${chip_root}/build/chip/chip_test_suite.gni")
 
-chip_test_suite_using_nltest("tests") {
+chip_test_suite("tests") {
   output_name = "libaccesstest"
-
   test_sources = [ "TestAccessControl.cpp" ]
 
-  cflags = [ "-Wconversion" ]
+  cflags = [
+    "-Wconversion",
+
+    # TODO: Fix compilation warnings and remove this
+    "-Wno-error=undef",
+    "-Wno-error=sign-compare",
```

Comment:
Is there anyway to have this get handled automatically by a public config of some sort for whatever pigweed headers involve these warnings, so we only need to fix one spot when we fix the headers?

At least I assume that's where the warnings are coming from...